wrangler-core/src/main/java/io/cdap/wrangler/utils/RowHelper.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.utils;
+
+import io.cdap.wrangler.api.Row;
+
+import java.util.List;
+
+/**
+ * Utility methods for {@link Row}
+ */
+public class RowHelper {
+  private RowHelper() {
+    throw new AssertionError("Cannot instantiate a static utility class");
+  }
+
+  /**
+   * Creates a uber record after iterating through all rows.
```

Comment:
nit: use "merged" instead of uber as it reflects the purpose more clear

Reply:
Done